scripts/analytics/legacy-ui-usage/README.md

```diff
@@ -0,0 +1,74 @@
+# Analytics scripts
+
+There are some python scrips to get insight into how users use the legacy Bichard UI.
+
+- `allocatedCases.py` can show the number of allocated cases per forces for a given time interval
+  - Pulls logs from CloudWatch that matches the search terms "allocated exception id:" or "allocated trigger id:"
+  - Queries the PG database to get the number of cases in that period
+  - Outputs the result in a spreadsheet and generates a bar chart using `plotly.express`
+- `search.py` can show the number of times different search boxes used
+  - Pulls search data from CloudWatch
+  - Queries the DynamoDB for `user.logged-in'` audit log events to get active users for a force in that period
+  - Outputs the result in a spreadsheet
+- `filters.py` can show the number of times different filters used
+  - Pulls search data from CloudWatch
+  - Queries the DynamoDB for `user.logged-in'` audit log events to get active users for a force in that period
+  - Outputs the result in a spreadsheet
+- `fetchHttpUsers.py` can show the number of users accessing Bichard via the old psn url
+  - Pulls data from CloudWatch
+  - By default it will query the last 3 hours, you can pass a optional parameter `HOUR_INTERVAL` to specify a different time period
+  - Queries the nginx logs for users still using the old url
+  - Outputs the result in a spreadsheet
+- `reportRunStats.py` can show the number of times reports generated per users/forces and combined for a given time interval
+  - Queries the DynamoDB for `report-run'` audit log events
+  - Queries the PG database to get the users' forces
+  - Requests forces data from the standing data repo to display full force names instead of codes
+  - Outputs the result in a spreadsheet
+
+### Usage
+
+Install packages:
+
+```
+pipenv install
+```
+
+Run `allocatedCases.py` script:
+
+- Before running this script connect to the production VPN and set the `CONNECTION_STRING` env variable for PG database.
```

Comment:
Might be helpful to link to [our Wiki page on specifically on getting the credentials](https://github.com/ministryofjustice/bichard7-next/wiki/Connecting-to-the-PostgreSQL-database#finding-the-credentials)?

Reply:
Resolved in [8876eb3](https://github.com/ministryofjustice/bichard7-next-core/pull/809/commits/8876eb357a6077a33d92db1788e44b2649d5c818)